src/concrete/ml/torch/compile.py

```diff
@@ -117,7 +120,8 @@ def build_quantized_module(
     # only work over shape of (1, ., .). For example, some reshape have newshape hardcoded based
     # on the inputset we sent in the NumpyModule.
     quantized_module = post_training_quant.quantize_module(*inputset_as_numpy_tuple)
-
+    if reduce_sum_copy:
```

Comment:
I would still add a fixme for https://github.com/zama-ai/concrete-ml-internal/issues/4127 here though !